src/app/hooks/useAPI.js

```diff
@@ -1,29 +1,30 @@
 import { useState, useEffect, useRef, useReducer } from 'react';
 import PropTypes from 'prop-types';
 
+import { FETCH_ACTIONS, REST_METHODS } from '~constants';
```

Comment:
avoid using `index.js` in `constants` and `utils` folder, the use for them is to store specific global utils or constants for the app, for your use case its something like: `constants/api.js` or store it in `hooks/constants.js`